### DISCUSSION TITLE
请问有PAG文件的规范声明文档吗？

### DISCUSSION THREAD

**Discussion by rubick24** [Tencent/libpag#47] [General] [answered]
类似https://www.w3.org/TR/2003/REC-PNG-20031110/ 这样的，可以让我们学习下pag文件结构或者实现自己的渲染器。

**Comment by domchen** [comment, marked as answer]
文件格式规范文档有的，我们近期整理一下会发布到官网。你也可以直接先从仓库里的 src/codec/ 目录下看起来，那边已经包含所有文件编解码的代码实现。

**Comment by th1nk3r-ing** [comment]
官网的文件格式没法下载, 能放到仓库中么

**Reply by domchen** [reply]
https://pag.io/docs/pag-spec.html  是部分浏览器问题，你右键点击另存为就下载下来了。
